_includes/twitter_logo.html

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22pt" height="22pt" viewBox="0 0 22 22" version="1.1">
-<g id="surface1">
-<rect x="0" y="0" width="22" height="22" style="fill:rgb(31.372549%,67.058824%,94.509804%);fill-opacity:1;stroke:none;"/>
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 2.914062 16.308594 C 4.378906 17.238281 6.117188 17.773438 7.976562 17.773438 C 13.207031 17.773438 17.449219 13.535156 17.449219 8.304688 C 17.449219 8.171875 17.445312 8.042969 17.441406 7.914062 C 17.628906 7.765625 18.800781 6.828125 19.085938 6.222656 C 19.085938 6.222656 18.136719 6.621094 17.207031 6.710938 C 17.203125 6.710938 17.203125 6.714844 17.199219 6.714844 C 17.199219 6.714844 17.203125 6.710938 17.203125 6.710938 C 17.289062 6.652344 18.488281 5.835938 18.652344 4.867188 C 18.652344 4.867188 17.980469 5.222656 17.039062 5.539062 C 16.882812 5.589844 16.71875 5.640625 16.550781 5.6875 C 15.945312 5.039062 15.082031 4.636719 14.125 4.636719 C 12.289062 4.636719 10.800781 6.125 10.800781 7.957031 C 10.800781 8.214844 10.832031 8.46875 10.886719 8.710938 C 10.632812 8.699219 6.726562 8.480469 4.066406 5.261719 C 4.066406 5.261719 2.480469 7.429688 5.015625 9.660156 C 5.015625 9.660156 4.242188 9.628906 3.578125 9.234375 C 3.578125 9.234375 3.335938 11.859375 6.203125 12.53125 C 6.203125 12.53125 5.640625 12.746094 4.738281 12.59375 C 4.738281 12.59375 5.242188 14.714844 7.792969 14.914062 C 7.792969 14.914062 5.777344 16.734375 2.914062 16.308594 Z M 2.914062 16.308594 "/>
-</g>
+<svg width="22pt" height="22pt" version="1.1" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox